File: Python/LeetCode_1480.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        len_list = len(nums)
        i=0
        sum_list=[]
        add_list=[]
        while i <len_list:
            
            sum_list.append(nums[i])
            add_list.append(sum(sum_list))
            i+=1
        return add_list

File: JavaScript/LeetCode_1480.js
var runningSum = function(nums) {
    var len_nums = nums.length;
    var i = 0;
    var sum_list =[];
    var add_list =[];
    while (i<len_nums){
        sum_list.push(nums[i]);
        var sum = 0
        for (var s=0;s<sum_list.length;s++){
            sum+=nums[s];
        }
        add_list.push(sum);
        i+=1;
    }

    return add_list ;    
};